blue_ai/envs/transient_goals.py

```diff
@@ -52,88 +57,114 @@ def __init__(
         self.transient_penalty = transient_penalty
         self.n_transient_obstacles = n_transient_obstacles
         self.transient_obstacles = transient_obstacles
-        self.replace_transient_obstacles = replace_transient_goals
```

Comment:
This is the only breaking change that happened; this is almost certainly just a mistake while doing Python strange class initiation syntax. None of the scripts I tried while testing broke from this.

While by no means comprehensive base grep search yielded no usages of the variable within the codebase (outside of the class itself)